account_manager/src/common.rs

```diff
@@ -19,3 +26,47 @@ pub fn base_wallet_dir(matches: &ArgMatches, arg: &'static str) -> Result<PathBu
         PathBuf::new().join(".lighthouse").join("wallets"),
     )
 }
+
+pub fn read_mnemonic_from_cli(
+    mnemonic_path: Option<PathBuf>,
+    stdin_password: bool,
+) -> Result<Mnemonic, String> {
+    eprintln!("");
+    println!("WARNING: IF YOU HAVE RUN THESE KEYS ON ANOTHER CLIENT, YOU RISK BEING SLASHED.");
+    eprintln!("");
+
+    let mnemonic = match mnemonic_path {
+        Some(path) => fs::read(&path)
+            .map_err(|e| format!("Unable to read {:?}: {:?}", path, e))
+            .map(|bytes| {
+                let bytes_no_newlines = strip_off_newlines(bytes);
```

Comment:
I think we should wrap this in `account_utils::PlainText` so these sensitive bytes are zero-ized on drop.

Reply:
Let me know if this I updated this correctly. I'm unsure how it works with the next line, which converts the phrase to `&str`

Reply:
Creating a `&str` should be OK because it just has a reference to the bytes owned by the `PlainText`, which will still get zeroized